api/hpa.go

```diff
@@ -39,7 +45,18 @@ func PutHpaResource(c *gin.Context) {
 		})
 		return
 	}
-	log.Info("Parse deployment succeeded")
+
+	err = scalingRequest.Validate()
+	if err != nil {
+		err := errors.Wrap(err, "Error parsing request:")
```

Comment:
@sagikazarmark can you please take a look? I don't think we need to wrap the error here.

Reply:
The scaling request internally uses the errors package from the standard library, so errors emitted by it do not have any stack trace, which might make sense in case of validation errors actually, so if that method only emits validation errors (and nothing like connection or timeout errors) then I think it's OK to use the errors package from the stdlib as validation errors are rather business errors than application errors.

For the same reason I agree with @baluchicken that annotating the error with a stack trace here doesn't add too much, since the error gets "handled" just a few lines later. If you want to add an extra message to the error, you can do so using `errors.WithMessage` (also, the extra colon at the end of the message is not necessary, error messages in Go are started with lower case by convention).